src/core/observer/dep.js

```diff
@@ -34,8 +33,7 @@ export default class Dep {
   }
 
   notify () {
-    // stabilize the subscriber list first
-    const subs = this.subs.slice()
+    const subs = Object.keys(this.subs).map(key => this.subs[key])
```

Comment:
Since I've gone down the rabbit hole of performance testing, here are some iteration tests.
http://jsben.ch/3s22e - and the winner is `Object.keys()` with a negative for loop, faster by 7% to a `keys -> map -> loop` solution

Reply:
Funny thing though, using object literal instead of `Object.create(null)` makes the values -> forEach version fastest almost by 200ms 🤔

Reply:
@yyx990803 Any more thoughts on this? So I could go ahead and refactor the iteration and finish it up.